Code/GraphMol/MolDraw2D/MolDraw2D.h

```diff
@@ -79,12 +91,26 @@ class MolDraw2D {
       const std::map<int, double> *highlight_radii = NULL, int confId = -1);
 
   virtual void drawMolecule(
+      const ROMol &mol, const std::string &legend,
+      const std::vector<int> *highlight_atoms = NULL,
+      const std::map<int, DrawColour> *highlight_map = NULL,
+      const std::map<int, double> *highlight_radii = NULL, int confId = -1);
+
+  virtual void drawMolecule(
```

Comment:
I understand. What about making a deprecation warning for the old API so we can at least plan for future sanity? :)